packages/cli/lib/validators/marketplaceValidators/theme/TemplateValidator.js

```diff
@@ -48,6 +48,7 @@ const VALIDATIONS_BY_TYPE = {
   proposal: { allowed: false },
   blog_listing: { allowed: true, label: true, screenshot: true },
   blog_post: { allowed: true, label: true, screenshot: true },
+  section: { allowed: true, label: false, screenshot: false },
```

Comment:
Is this needed since sections have their own validator?

Reply:
Nice catch. I ended up adding a `TEMPLATE_IGNORE_LIST` that ignores any templateType that we want to ignore. Currently we're just ignoring `section`.